### PR TITLE
set guzzle requirement to ^7

### PR DESCRIPTION
**Pull request by spencer95**

### CODE REVIEW

**Comment by spencer95** [comment]
guzzle 7

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "license": "MIT",
     "minimum-stability": "stable",
     "require": {
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^7",
```
Comment (lammensj) [inline composer.json:7]: To be backwards compatible, I would advice to use `^6.3 || ^7`

**Review by lammensj — CHANGES_REQUESTED** [review]

**Comment by clairemistry** [comment]
I really need this change too in my  Drupal 10 project.  Could we get this merged in? Thank you so much

**Comment by lammensj** [comment]
I don't work at this company anymore, hence I don't have write-access. I'll reach out to my former colleagues, keep you posted.

**Comment by lammensj** [comment]
Done!

**Pull request closed without merging**